src/levels.rs

```diff
@@ -209,10 +209,12 @@ pub const COMP_INTER_AVG: CompInterType = 2;
 pub const COMP_INTER_WEIGHTED_AVG: CompInterType = 1;
 pub const COMP_INTER_NONE: CompInterType = 0;
 
-pub type InterIntraType = u8;
-pub const INTER_INTRA_WEDGE: InterIntraType = 2;
-pub const INTER_INTRA_BLEND: InterIntraType = 1;
-pub const INTER_INTRA_NONE: InterIntraType = 0;
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum InterIntraType {
+    None = 0,
+    Blend = 1,
+    Wedge = 2,
+}
```

Comment:
What if we removed `InterIntraType::None` and used `Option::None` instead?  It'd make a bunch of the `match`es exhaustive since the initial `None` check can be done using `if let Some`.

Reply:
yes that's nicer